apps/server/src/modules/video-conference/config/bbb-create.config.ts

```diff
@@ -18,6 +18,7 @@ export class BBBCreateConfig extends BBBBaseMeetingConfig {
 		this.moderatorPW = config.moderatorPW;
 		this.attendeePW = config.attendeePW;
 		this.allowModsToUnmuteUsers = config.allowModsToUnmuteUsers;
+		this['meta_bbb-origin-server-name'] = config['meta_bbb-origin-server-name'];
```

Comment:
change in file looks good but the /* istanbul ignore next */ on top of the constructure should be removed in one of the next steps.

Reply:
done